src/events/guild-create.ts

```diff
@@ -1,14 +1,30 @@
-import {Guild, TextChannel, Message, MessageReaction, User} from 'discord.js';
+import {Guild, TextChannel, Message, MessageReaction, User, ApplicationCommandData,
+} from 'discord.js';
 import emoji from 'node-emoji';
 import pEvent from 'p-event';
 import {Settings} from '../models/index.js';
 import {chunk} from '../utils/arrays.js';
+import container from '../inversify.config.js';
+import Command from '../commands';
+import {TYPES} from '../types.js';
+import Config from '../services/config.js';
 
 const DEFAULT_PREFIX = '!';
 
 export default async (guild: Guild): Promise<void> => {
   await Settings.upsert({guildId: guild.id, prefix: DEFAULT_PREFIX});
 
+  const config = container.get<Config>(TYPES.Config);
+
+  // Setup slash commands
+  if (config.IS_PRODUCTION) {
```

Comment:
```suggestion
  if (!config.IS_PRODUCTION) {
```

I think? If I'm understanding correctly.

Reply:
Well, this was a bad typo, corrected